gef-ui/server/utils/helpers.js

```diff
@@ -79,21 +83,125 @@ const validationErrorHandler = (errs, href = '') => {
   };
 };
 
+/*
+   Maps through facilities to check for changedToIssued to be true
+   if true, adds to array and returns array
+*/
+const facilitiesChangedToIssuedAsArray = (application) => {
+  const hasChanged = [];
+  application.facilities.items.map((facility) => {
+    if (facility.details.changedToIssued === true) {
```

Comment:
if (facility.details.changedToIssued === true) can be if (facility.details.changedToIssued)?

Reply:
done